Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -147,7 +147,7 @@ public class ChatChannelController: DataController, DelegateCallable, DataStoreP
     /// It is `true` if the channel typing events are enabled as well as the user privacy settings.
     internal var shouldSendTypingEvents: Bool {
         /// Ignore if user typing indicators privacy settings are disabled. By default, they are enabled.
-        let currentUserPrivacySettings = client.currentUserController().currentUser?.privacySettings
+        let currentUserPrivacySettings = client.sharedCurrentUserController.currentUser?.privacySettings
```

Comment:
When creating the `ChannelController,` we can probably also create a currentUserController and maintain a reference to it. 🤔 
```suggestion
        let currentUserPrivacySettings = self.currentUserController.currentUser?.privacySettings
```